index.html

```diff
@@ -6,6 +6,6 @@
     <title>CSD120 Jason Leveille</title>
 </head>
 <body>
-    
-</body> "Hello world!"
+    Hello world!
+</body>
```

Comment:
The content of an element must go **between** the opening and closing tags
Typically, if I give you an instruction like add the text `"Hello, world!"` to some element, you don't need to include the quotation marks